### DISCUSSION TITLE
Notify timeout seems not obeying its configuration

### DISCUSSION THREAD

**Discussion by nmsobri** [LazyVim/LazyVim#485] [Q&A]
I try to decrease notify timeout, yet it still doesn't take any effect

here is my Notify config:

```
{
    "rcarriga/nvim-notify",
    keys = {{
        "<leader>un",
        function()
            require("notify").dismiss({
                silent = true,
                pending = true
            })
        end,
        desc = "Delete all Notifications"
    }},
    opts = {
        timeout = 1000,
        max_height = function()
            return math.floor(vim.o.lines * 0.75)
        end,
        max_width = function()
            return math.floor(vim.o.columns * 0.75)
        end
    }
}
```

Im expecting notify will only show the message for 1 second, yet it take more than that.. Please help

**Comment by folke** [comment]
The timeout does work, but it's not as instant as the timeout in some cases I noticed myself.

Try setting it to 100 and then to 10000. You will see that it definitely does something. You can also ask in the nvim-notify repo.

Apart from that, Noice explicitly keeps some notifs shown when noice needs to show a e message during a blocking event. That is to make sure the user would not miss a notif. For example, when the cmdline is open, every notification will stay on the screen till you close the cmdline. Only then will the nvim-notify timeout start.

**Reply by nmsobri** [reply]
i set the timout to 100, it still open for 3 seconds.. i don't know whats wrong.. is my configuration correct?
